PlayCover/Utils/PlayTools.swift

```diff
@@ -255,32 +538,33 @@ class PlayTools {
             }
         }
 	}
+}
 
-    private static func binPath(_ bin: String) throws -> URL {
-        URL(fileURLWithPath: try shell.sh("which \(bin)", print: false).trimmingCharacters(in: .newlines))
-    }
-
-    private static var vtool: URL {
-        get throws {
-            try binPath("vtool")
-        }
-    }
-
-    private static var otool: URL {
-        get throws {
-            try binPath("otool")
-        }
+extension URL {
```

Comment:
It would be best to move URL and String extensions into the URLExtensions file along with the other URL and String extensions.